src/js/tabs/pid_tuning.js

```diff
@@ -2494,12 +2493,12 @@ pid_tuning.initialize = function (callback) {
         self.updating = false;
 
         // enable RC data pulling for rates preview
-        GUI.interval_add('receiver_pull', self.getReceiverData, true);
+        GUI.interval_add('receiver_pull', self.getReceiverData, 250, true);
```

Comment:
Why we need a timer here now @haslinghuis ?

Reply:
Assumed it was missing a timer in the interval code?

Reply:
It was missing a timer (bug) in the interval code. Without type checking like TS, JS did allow it to run.

Function is defined as 
```
GuiControl.prototype.interval_add = function (name, code, interval, first) {
```